src/main/java/org/reactome/addlinks/db/ReferenceObjectCache.java

```diff
@@ -106,183 +106,217 @@ private static synchronized void buildReferenceCaches(String className, Map<Stri
 				logger.error("Could not get the \"{}\" attribute for {}. Reason: {}", identifierAttribute, referenceObject, e.getMessage());
 			}
 
-			// ReferenceMolecules and DatabaseIdentifiers do not have associated species.
-			if ( !className.equals(ReactomeJavaConstants.ReferenceMolecule) && !className.equals(ReactomeJavaConstants.DatabaseIdentifier) )
+			// ReferenceMolecules and DatabaseIdentifiers and ReferenceTherapeutics do not have associated species.
+			if ( !className.equals(ReactomeJavaConstants.ReferenceMolecule) && !className.equals(ReactomeJavaConstants.DatabaseIdentifier) && !className.equals(ReactomeJavaConstants.ReferenceTherapeutic))
```

Comment:
If this block is only trying to filter out classes where species isn't valid, you can also use `instance.getSchemClass().isValidAttribute(ReactomeJavaConstants.species)`

Reply:
If there is a `GKInstance` available, I should say

Reply:
I liked listing them explicitly, to make it more clear, but if I have to add any more, I might switch to the `isValidAttribute` method.